test/xref2/github_issue_944.t/run.t

```diff
@@ -0,0 +1,8 @@
+A quick test to repro the issue found in #944
+
+  $ ocamlc -bin-annot -c foo.mli
+
+  $ odoc compile foo.cmti
+  $ odoc link foo.odoc
```

Comment:
You can add `--enable-missing-root-warning` to ensures that there's still an error. Perhaps use module names that are not in the stdlib in the `.mli` file.

Reply:
Done